plc4j/drivers/s7/src/main/java/org/apache/plc4x/java/s7/readwrite/protocol/S7ProtocolLogic.java

```diff
@@ -69,7 +72,7 @@
  */
 public class S7ProtocolLogic extends Plc4xProtocolBase<TPKTPacket> {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(S7ProtocolLogic.class);
```

Comment:
as it's a static, I would prefer the LOGGER ... or make it non-static, then use the lowercase version.

Reply:
Got spot, I did not notice it was static. AFAIR slf4j and modern log4j (even 1.2) recommendation is to use final without static classifier.